From 80c4b6f391abefaf7ba336ea9f602c92baec15b5 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sun, 30 Mar 2014 14:25:11 +0200
Subject: [PATCH] CMakeLists: check only for libsystemd pkg

* compat libs were merged into one libsystemd library in systemd-210

Change-Id: Ib9ff696e1332335887a127d64e64e5b59fc91e11
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cd27b5b..a7d2fdc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,7 @@ pkg_check_modules(PMLOGLIB REQUIRED PmLogLib>=3.0.2)
 include_directories(${PMLOGLIB_INCLUDE_DIRS})
 webos_add_compiler_flags(ALL ${PMLOGLIB_CFLAGS_OTHER})
 
-pkg_check_modules(SYSTEMD_DAEMON REQUIRED libsystemd-daemon)
+pkg_check_modules(SYSTEMD_DAEMON REQUIRED libsystemd)
 include_directories(${SYSTEMD_DAEMON_INCLUDE_DIRS})
 webos_add_compiler_flags(ALL ${SYSTEMD_DAEMON_CFLAGS_OTHER})
 
-- 
1.9.1

